### PR TITLE
New package: conntrack-tools-1.4.3 + dependencies

### PR DESCRIPTION
**Pull request by ebfe**

### CODE REVIEW

File: srcpkgs/libnetfilter_queue/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Template file for 'libnetfilter_queue'
+pkgname=libnetfilter_queue
+version=1.0.2
+revision=2
```
Comment (dominikh) [inline srcpkgs/libnetfilter_queue/template]: revision=1

File: srcpkgs/conntrack-tools/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'conntrack-tools'
+pkgname=conntrack-tools
+version=1.4.3
+revision=2
```
Comment (dominikh) [inline srcpkgs/conntrack-tools/template]: revision=1

**Comment by ebfe** [comment]
Squashed the musl-fix commits, and reset revisions to 1.

File: srcpkgs/libnetfilter_cthelper/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+makedepends="libmnl-devel"
+short_desc="Interface to the connection tracking user-space helper infrastructure"
+maintainer="Michael Gehring <mg@ebfe.org>"
+license="GPL-2"
+homepage="http://www.netfilter.org/projects/libnetfilter_cthelper/index.html"
+distfiles="http://www.netfilter.org/projects/libnetfilter_cthelper/files/libnetfilter_cthelper-${version}.tar.bz2"
+checksum=07618e71c4d9a6b6b3dc1986540486ee310a9838ba754926c7d14a17d8fccf3d
+
+libnetfilter_cthelper-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
```
Comment (xtraeme) [inline srcpkgs/libnetfilter_cthelper/template]: `depends="${makedepends} ${sourcepkg}>=${version}_${revision}"`

File: srcpkgs/libnetfilter_cttimeout/template [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+makedepends="libmnl-devel"
+short_desc="Interface to the fine-grain connection tracking timeout infrastructure"
+maintainer="Michael Gehring <mg@ebfe.org>"
+license="GPL-2"
+homepage="http://www.netfilter.org/projects/libnetfilter_cttimeout/index.html"
+distfiles="http://www.netfilter.org/projects/libnetfilter_cttimeout/files/libnetfilter_cttimeout-${version}.tar.bz2"
+checksum=aeab12754f557cba3ce2950a2029963d817490df7edb49880008b34d7ff8feba
+
+libnetfilter_cttimeout-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
```
Comment (xtraeme) [inline srcpkgs/libnetfilter_cttimeout/template]: `depends="${makedepends} ${sourcepkg}>=${version}_${revision}"`

File: srcpkgs/libnetfilter_queue/template [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+CFLAGS="-D_GNU_SOURCE"
+
+pre_configure() {
+	sed -i '15i#include <sys/time.h>' include/libnetfilter_queue/libnetfilter_queue.h
+	sed -i 's/u_int\([0-9]*\)_t/uint\1_t/g' $(find -name '*.[ch]')
+}
+
+libnetfilter_queue-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
```
Comment (xtraeme) [inline srcpkgs/libnetfilter_queue/template]: `depends="${makedepends} ${sourcepkg}>=${version}_${revision}"`

Comment (ebfe) [inline srcpkgs/libnetfilter_queue/template]: > `depends="${makedepends} ${sourcepkg}>=${version}_${revision}"`

All fixed.

**Pull request merged**